book/controller.rst

```diff
@@ -803,6 +803,29 @@ Just like when creating a controller for a route, the order of the arguments of
 order of the arguments, Symfony will still pass the correct value to each
 variable.
 
+Checking the Validity of a CSRF Token
+-------------------------------------
+
+Sometimes you want to use CSRF protection in an action where you don't want to use the
+Symfony Form component.
+
+If, for example, you're doing a DELETE action, you can use the
+:method:`Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller::isCsrfTokenValid`
+method to check the CSRF token::
+
+    if ($this->isCsrfTokenValid('token_id', $submittedToken)) {
+        // ... do something, like deleting an object
+    }
+
+.. versionadded:: 2.6
+    The ``isCsrfTokenValid()`` shortcut method was introduced in Symfony 2.6.
+
+It is equivalent to executing the following code::
```

Comment:
you should indent this (and put this directly after the previous sentence). This way, everything will be included in the versionadded box. Again, this can be done after the merge (please note that you can just add commits to the current branch, no need to create a new PR)